lib/davinci_pas_test_kit/generator/templates/suite.rb.erb

```diff
@@ -76,6 +77,24 @@ module DaVinciPASTestKit
       #   request.query_parameters['token']
       # end
 
+      group 'Subscription Setup' do
```

Comment:
since there is input / output passing between these tests, this should be `run_as_group`

Reply:
https://github.com/inferno-framework/davinci-pas-test-kit/pull/31/commits/439df49a8aefb04f04d53630dbc1f54dc05459d3